### PR TITLE
Check if we actually have env vars set

### PR DESCRIPTION
**Pull request by CorneliusIV**
This connects to #24

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -29,9 +29,35 @@ type DockerInfo struct {
...
+	Config.BUILDLOCATION = os.Getenv("BUILD_LOCATION")
+	if Config.BUILDLOCATION == "" {
+		logging.LogToFile("Missing environment variable BUILD_LOCATION")
+		panic("Missing environment variable BUILD_LOCATION")
+
+	}
+	Config.FRANKLINAPIURL = os.Getenv("API_URL")
+	if Config.FRANKLINAPIURL == "" {
+		logging.LogToFile("Missing environment variable API_URL")
+		panic("Missing environment variable API_URL")
```
Comment (CorneliusIV) [inline api.go:51]: I don't love having to explicitly write these out, but I couldn't think of a cleaner way to make sure that these are required and give you and throw an error early enough.

Comment (JulianGindi) [inline api.go:51]: Agreed, but this doesn't seem to terrible to me. Another option is to include it in the Makefile.

**Comment by oudeismetis** [comment]
:+1:

**Comment by JulianGindi** [comment]
:+1:

**Pull request closed without merging**